### DISCUSSION TITLE
2024/01/14/20240114_updatelog/

### DISCUSSION THREAD

**Discussion by giscus** [cyclinbox/cyclinbox.github.io#11] [General]
# 2024/01/14/20240114_updatelog/

本周更新了一篇技术文章。此外，对网站进行了一些调整，新增了文章搜索的功能。

https://wz.anoms.top/2024/01/14/20240114_updatelog/

**Comment by Hazelandinstant** [comment]
我也想拥有这样的工作环境！！！
我最近也感觉现在的时光好像有点快，我有点记不住发生了什么，好日子、快乐的日子都在过去。 可是年终总结的时候，我发现今年尝试了很多自我突破的事情，但不知道为什么自己似乎总在忽略掉。
我以前听说过一个理论：孩子之所以觉得每一天、每一年都很漫长，是因为他们总共在这世上也没几年，一天时间/一年时间，自然每一天、每一年都很有意义；而对于成人来说，一天的时间只是人生几十年中的一天，所以越长大越觉得时光匆匆。
另外，IBP的生活严格来说只过去了一年半 :-)
今年最大的收获就是遇见了优秀的人吧，虽然“斯人若彩虹，遇上方知有”有些不恰当，但是，遇见并能发生交集，也确实足够幸运，足够有收获了，也稍微缓解了一下对于时间的焦虑。

**Reply by cyclinbox** [reply]
要是从刚到IBP的时候开始算，差不多就是三年了 :-P （一年半和三年，间接反映时间确实过得好快）
时间长短的那个理论很有趣，但这个理论中，最打动我的也许是赋予时间的“意义”。如果现在我们依然可以做一些挑战自我的、有意义的事情，也许时间就变得不那么快、变得更充实了。所以还是加油——给时间以意义，而非给意义以时间
还有还有，遇上优秀的人并能发生交集，这真是一件幸运的事情（好羡慕）
愿所有人都可以找到属于自己的幸福~

**Comment by EamonNKU** [comment]
踩踩，时间过得好慢。躺床上刷变了各大能想到的app，甚至刷了一圈远古博客，一天还是没结束

**Reply by cyclinbox** [reply]
awa，话说你们已经放假了嘛

**Reply by cyclinbox** [reply]
卧槽哥你的主页也改版了，感觉变化好大（但是不知道为啥只能ip地址访问，url访问会出错）

**Reply by EamonNKU** [reply]
你为什么会知道我的主页ip地址？？神奇

**Reply by EamonNKU** [reply]
下周五最后一次组会

**Reply by cyclinbox** [reply]
Windows上有个特别好用的指令叫做`ping`，它的使用方法大体是这个样子：

```text
C:\>ping fanyiming.life

Pinging fanyiming.life [xx.xxx.xxx.x] with 32 bytes of data:
Reply from xx.xxx.xxx.x: bytes=32 time=32ms TTL=49
Reply from xx.xxx.xxx.x: bytes=32 time=30ms TTL=49
Reply from xx.xxx.xxx.x: bytes=32 time=30ms TTL=49
Reply from xx.xxx.xxx.x: bytes=32 time=32ms TTL=49

Ping statistics for xx.xxx.xxx.x:
    Packets: Sent = 4, Received = 4, Lost = 0 (0% loss),
Approximate round trip times in milli-seconds:
    Minimum = 30ms, Maximum = 32ms, Average = 31ms
```

（上面的IP地址已经帮你打码了）

总之你在cmd里面跑一下，就能看到URL解析到的IP地址~

**Reply by EamonNKU** [reply]
tql

**Reply by cyclinbox** [reply]
呜呜
